langchain/vectorstores/pgvector.py

```diff
@@ -291,8 +291,15 @@ def similarity_search_with_score_by_vector(
         if filter is not None:
             filter_clauses = []
             for key, value in filter.items():
-                filter_by_metadata = EmbeddingStore.cmetadata[key].astext == str(value)
-                filter_clauses.append(filter_by_metadata)
+                IN = "IN"
```

Comment:
could we use a lower case `in` -- I think it's responsible to use `snake_case` given that we're working in python.

@hwchase17 / @agola11 is there a common representation for filters?

Reply:
switched to lower case, and made the matching case insensitive.

mainly made this choice to match with postgres, which allows any case.